zeebe/snapshot/src/test/java/io/camunda/zeebe/snapshots/SnapshotCopyUtil.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.zeebe.snapshots;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.function.BiConsumer;
+
+public final class SnapshotCopyUtil {
+
+  /**
+   * @return a {@link BiConsumer }that copies all files from the path at first argument to the path
+   *     at second argument. Used to copy all files of a snapshot from one place to another to test
+   *     the copy
+   */
+  public static BiConsumer<Path, Path> copyAllFiles() {
```

Comment:
:question:/:wrench: Why is this a method returning a lambda? To me this looks like an automatic refactoring gone wrong. This should just be a method taking source and target arguments and then all uses should just use the method reference instead of calling `copyAllFiles`.